docs/docusaurus.config.js

```diff
@@ -156,6 +156,27 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        appId: 'NTSX40VZB8',
+
+        // Public API key, safe to commit
+        apiKey: '4269c726c2fea4e6cddfeb9a21cd3d4e',
+
+        indexName: 'kurtosis',
+
+        contextualSearch: true,
+
+        externalUrlRegex: 'external\\.com|domain\\.com',
```

Comment:
what does this dO?

Reply:
`// Optional: Specify domains where the navigation should occur through window.location instead on history.push. Useful when our Algolia config crawls multiple documentation sites and we want to navigate with window.location.href to them.`

Reply:
ended up removing cause I don't know if the options needed, can add back later if needed.